### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by paraflu** [Tobbe/CppIEEmbed#3]
Hi I've tried your class but I've some memory issue using multiple html page... have you worked more on this code?

**Comment by Tobbe** [comment]
No, I haven't worked any more on the code. I don't have a Windows environment to code in anymore.
